apps/frontend/src/modules/auth/slices/auth.slice.ts

```diff
@@ -50,6 +57,21 @@ const { actions, name, reducer } = createSlice({
 		builder.addCase(getAuthenticatedUser.rejected, (state) => {
 			state.dataStatus = DataStatus.REJECTED;
 		});
+
+		builder.addCase(requestResetPassword.fulfilled, () => {
+			notification.success(NotificationMessage.LINK_SENT);
```

Comment:
Should be done in action

Reply:
Because it is a side effect, fair enough